manifests/install.pp

```diff
@@ -18,4 +18,18 @@
     ensure => present,
   }
 
+  selinux::port { 'pulpcore-api-port':
```

Comment:
I'd consider this config because it needs the install for the package to be installed. We may also need to ensure `pulpcore-selinux` is installed (in `install.pp`)

Reply:
@ekohl I've moved those declarations into `config.pp`

About the package, I found this: https://github.com/theforeman/foreman-packaging/pull/4648/files#r382513097

From that discussion it seems that we really can't rely on `python3-pulpcore` to install `pulpcore-selinux` so I've declared the package as well.